apps/console/src/features/identity-providers/components/forms/components/common-pluggable-component-form.tsx

```diff
@@ -310,21 +355,32 @@ export const CommonPluggableComponentForm: FunctionComponent<CommonPluggableComp
                 disableField("SignatureAlgorithm");
                 disableField("DigestAlgorithm");
             }
-
         }
+    };
+
+    const changeMode = (key: string, values: Map<string, FormValue>) => {
+        const selectedMode: FormValue = values.get("SelectMode");
+
+        if (key === "SelectMode") {
 
+            if (selectedMode === "Manual Configuration") {
```

Comment:
Shall we use an enum to store the possible values for the `selectMode`

Reply:
Improved with 769cf8391b5176fdb1cbee9cf4cc3fe82c8057d4.